packages/context/src/context.ts

```diff
@@ -11,8 +11,9 @@ export type Constructor<T> = new(...args) => T;
 export class Context {
   private registry: Map<string, Binding>;
 
-  constructor() {
+  constructor(private _parent?: Context) {
     this.registry = new Map();
```

Comment:
On the second thought, let's move the unique tracing id out of scope of this patch - see https://github.com/strongloop/loopback-next/issues/209